src/main/java/org/mskcc/cbio/oncokb/service/dto/CompanyDTO.java

```diff
@@ -41,9 +41,11 @@ public class CompanyDTO implements Serializable {
 
     @NotEmpty
     private Set<String> companyDomains = new HashSet<>();
+    @Lob
+    private String additionalInfo;
```

Comment:
We should have an AdditionalInfo class to host all properties, similar to AdditionalInfoDTO under useradditionalinfo